app/components/works/keywords_component.html.erb

```diff
@@ -1,5 +1,5 @@
-<div class="keywords" data-controller="keywords" data-action="focusout->keywords#close">
-  <div class="keywords-container <%= 'is-invalid' if errors.present? %>" data-target="keywords.container" data-action="click->keywords#open">
+<div class="keywords" data-controller="keywords" data-action="focusout->keywords#close" data-target="progress.keywordsField">
```

Comment:
Allow progress controller to target the keywords containing div. Progress controller is scoped to the entire work form, so we can use this pattern and the progress controller for more fields in the future if we wish.